GitUI/Editor/FileViewer.cs

```diff
@@ -455,7 +455,7 @@ string GetFileText()
 
                 if (!File.Exists(path))
                 {
-                    return null;
+                    return $"File {path} does not exist";
```

Comment:
I get this text in combination with #7831 for an unstaged new file:
"File F:\Build\gitextensions3\- Kopie.editorconfig does not exist"

Reply:
The part of the PR that was removed fixed that, but there may have been sideeffects so it was dropped

> Un/staging works well. Edit, Open with..., Delete are missing in the context menu.

Also related to the file resolver